src/super_gradients/training/models/detection_models/yolo_base.py

```diff
@@ -408,6 +421,29 @@ def __init__(self, backbone: Type[nn.Module], arch_params: HpmStruct, initialize
             self._head = YoloHead(self.arch_params)
             self._initialize_module()
 
+        self._image_processor = ComposeProcessing(
+            [
+                DetectionRescale((640, 640), keep_aspect_ratio=True),
+                DetectionSidePadding((640, 640), 114),
+                ImagePermute((2, 0, 1)),
+            ]
+        )
+        self._class_names = COCO_DETECTION_CLASSES_LIST
```

Comment:
We should not hardcode default _class_names.

Reply:
Changes this and _image_processor logic in f3db9559cff93e54e3a403dd2eae12d4bb88f908